packages/shared/styles/components/SfNotification.scss

```
@@ -0,0 +1,93 @@
+@import '../variables';
+$notification__message-padding-left: 0.625rem !default;
+$notification--info-color: $c-dark-primary !default;
+$notification--warning-color: #d12727 !default;
+$notification--alert-color: #0468db !default;
+$notification--success-color: #128830 !default;
+$notification-mobile-padding: $spacer-big !default;
```

```suggestion
$notification-padding: $spacer-big !default;
```
it's just a padding ;)

---

changed